### DISCUSSION TITLE
How does one un-float a floating window?

### DISCUSSION THREAD

**Discussion by raiguard** [qtile/qtile#3432] [Q&A] [answered]
Title. I searched through the docs, issues, and discussions but turned up a blank. How can one un-float a window so it is re-introduced into the tiling? I apologize if I'm just blind.

![image](https://user-images.githubusercontent.com/3515394/158866832-0eae7846-9a08-4f3c-852c-fe6418e8ba5f.png)

**Comment by elParaguayo** [comment, marked as answer]
`lazy.window.toggle_floating()`

**Reply by raiguard** [reply]
:facepalm: Thanks, I really am blind.

`Key([mod], "f", lazy.window.toggle_floating()),`
